burnman/processchemistry.py

```diff
@@ -59,7 +59,7 @@ def dictionarize_formula(formula):
     return f
 
 def sum_formulae(formulae, amounts=None):
-    if amounts == None:
+    if amounts is None:
```

Comment:
I cannot comment on non-changes.... but zip() in line 143 needs to a list() around it for python3.